projects/plugins/crm/src/js/components/automations-admin/types.tsx

```diff
@@ -0,0 +1,37 @@
+export type Trigger = {
+	slug: string;
+	title: string;
+	description?: string;
+	category: Category;
+};
+
+export type Step = {
+	attributes: string[];
+	nextStep?: Step;
+	slug: string;
+	title: string;
+	description: string;
+	type: Type;
+	category: Category;
+	allowedTriggers: Trigger[];
+};
+
+export type Action = Step;
+
+export type Condition = Step;
+
+export type Workflow = {
+	id: number;
+	name: string;
+	description: string;
+	category: Category;
+	triggers: Trigger[];
+	initial_step: Step;
+	active: boolean;
+	version: number;
+	added: string;
+};
+
+export type Type = 'contacts';
```

Comment:
Why is `Type` defined as `'contacts'` ?

Reply:
The only type I could find in the code was `contacts` so I just stuck with that and planned to add others once they are known.